prepare-vms/scripts/trainer-cli

```diff
@@ -466,7 +464,7 @@ run_cli() {
         ;;
     *)
         echo "
-trainer COMMAND [n-instances|tag]
+./trainer COMMAND [n-instances|tag|settings_file]
```

Comment:
It should be `./trainer <command> [n-instances|tag] [settings/file.yaml]`, I believe